client/lib/importer/actions.js

```diff
@@ -1,303 +0,0 @@
-/**
```

Comment:
Moving this to `state/imports` would make a big PR on its own. Perhaps it would be a good way to start separating those changes into multiple PRs?

Reply:
You mean moving the Flux store to `state/imports` as is and then modify it there?

The `actions.js` file gets a rather major rewrite here -- moving to thunks, removing all Flux dispatches, removing redundant actions...

Reply:
I was thinking of creating all the new actions / thunks in a PR 1, updating the usages from flux to redux in PR 2, and removing old flux in PR 3. But maybe that's not as straightforward as it sounds for this particular instance...

Reply:
OK, I'll try to start splitting smaller PR, including the Flux => Redux workflow you just described.

Anyway, the first next step now should be landing #27396. It has some refactorings that are quite straightforward and should be easy to review.

Reply:
On it 👍